plugin/trino-iceberg/src/main/java/io/trino/plugin/iceberg/catalog/hms/TrinoHiveCatalog.java

```diff
@@ -286,22 +281,9 @@ public Transaction newCreateTableTransaction(
     public List<SchemaTableName> listTables(ConnectorSession session, Optional<String> namespace)
     {
         ImmutableSet.Builder<SchemaTableName> tablesListBuilder = ImmutableSet.builder();
-        listNamespaces(session, namespace)
```

Comment:
For context, this is the initial commit which added the filtering of Iceberg tables in the listing:

https://github.com/trinodb/trino/commit/a2ea77b6f4f0f208b00ca4c6d2e99eb6814538b8

Reply:
Even farther back https://github.com/trinodb/trino/pull/1354